### PR TITLE
DOCS-2004: TrueCommand 2.0-Beta Release Notes

### PR DESCRIPTION
**Pull request by jervin777**
Initial content

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* Faster Database
+* Real-Time Stats
+* Storage Navigator
+* iSCSI Manager
+* Improved System-Down alerts
+* Secure Web-Proxy for Cloud
+* TrueNAS SCALE cluster UI
+
+
+[TrueCommand](https://www.truenas.com/truecommand/) was launched in 2019 as a single pane of glass management system for TrueNAS fleets. [TrueCommand 1.3](https://www.truenas.com/docs/truecommand/) has been adopted by well over three thousand organizations to manage their NAS fleets. TrueCommand is deployed as a Container, as a VM, or as a Cloud service, aka [TrueCommand Cloud](https://www.ixsystems.com/blog/truenas-with-truecommand-cloud/). As promised, TrueCommand remains free for users with less than 50 drives and is very affordable for larger organizations, where TrueCommand coordinates a team of people to manage many systems on a 24 x 365 basis.
```
Comment (Mrt134) [inline content/ReleaseNotes/TrueCommand/2.0-Beta.md:21]: Let's set the 1.3 link as a relref to the 1.3 release notes: 
```suggestion
[TrueCommand](https://www.truenas.com/truecommand/) was launched in 2019 as a single pane of glass management system for TrueNAS fleets. [TrueCommand 1.3]({{< relref "/ReleaseNotes/TrueCommand/1.3.md" >}}) has been adopted by well over three thousand organizations to manage their NAS fleets. TrueCommand is deployed as a Container, as a VM, or as the [TrueCommand Cloud](https://www.ixsystems.com/blog/truenas-with-truecommand-cloud/) service. TrueCommand is free for users with less than 50 drives and is very affordable for larger organizations, where TrueCommand coordinates a team of people to manage many systems on a 24 x 365 basis.
```

Comment (Mrt134) [reply content/ReleaseNotes/TrueCommand/2.0-Beta.md:21]: Added a couple text tweak suggestions too

**Comment by Mrt134** [comment]
Adding Joe and Kris for final approval

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Login to the [TrueCommand Account Portal](https://portal.ixsystems.com) for downloads, documentation links, and licensing options.
+For storage clusters with more than 50 disks, the account portal also offers a *free 60-day trial license with unlimited disks*.
+ 
+## To Update to this Release
+ 
+**Docker:** Re-run `docker pull ixsystems/truecommand:2.0_beta` to fetch this version of TrueCommand, and then restart your docker instance.
+ 
+{{< hint warning >}}
+**Pre-Release Testing Version**
+This is a BETA release and is not usable for production systems.
```
Comment (kmoore134) [inline content/ReleaseNotes/TrueCommand/2.0-Beta.md:114]: "not recommended for production systems"

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+* Improved System-Down alerts
+* Secure Web-Proxy for Cloud
+* TrueNAS SCALE cluster UI
+
+
+[TrueCommand](https://www.truenas.com/truecommand/) was launched in 2019 as a single pane of glass management system for TrueNAS fleets. [TrueCommand 1.3]({{< relref "/ReleaseNotes/TrueCommand/1.3.md" >}}) has been adopted by well over three thousand organizations to manage their NAS fleets. TrueCommand is deployed as a Container, as a VM, or as the [TrueCommand Cloud](https://www.ixsystems.com/blog/truenas-with-truecommand-cloud/) service. TrueCommand is free for users with less than 50 drives and is very affordable for larger organizations, where TrueCommand coordinates a team of people to manage many systems on a 24 x 365 basis.
+
+## 2.0-Beta Changelog
+
+### New Feature
```
Comment (kmoore134) [inline content/ReleaseNotes/TrueCommand/2.0-Beta.md:25]: Do we want to list any of the large epic tickets here for Cluster management, iSCSI manager or other big features?

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                      <td></td>
+                  </tr>
+                  <tr>
+                      <td></td>
+                      <td>Cluster Volumes.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
+                  </tr>
+                      <td></td>
+                      <td>iSCSI Management.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
```
Comment (kmoore134) [inline content/ReleaseNotes/TrueCommand/2.0-Beta.md:96]: and is still being polished. Please report bugs <here>

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                <tbody>
+                   <tr>
+                      <td><a href="https://jira.ixsystems.com/browse/NAS-110413" target="_blank">NAS-110413</a></td>
+                      <td>The UI is misreading the schedule for a Snapshot Task from True Command.</td>
+                      <td></td>
+                  </tr>
+                  <tr>
+                      <td></td>
+                      <td>Cluster Volumes.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
```
Comment (kmoore134) [inline content/ReleaseNotes/TrueCommand/2.0-Beta.md:92]: and is still being polished. Please report bugs <here>

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md (hunk 3 above)
Comment (Mrt134) [reply content/ReleaseNotes/TrueCommand/2.0-Beta.md:25]: this is a straight port of the "New Feature" type tickets from https://jira.ixsystems.com/secure/ReleaseNote.jspa?projectId=10300&version=12906 - do you want the Epic section combined in here or done as a separate section?

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md (hunk 4 above)
Comment (jervin777) [reply content/ReleaseNotes/TrueCommand/2.0-Beta.md]: Added

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md (hunk 5 above)
Comment (jervin777) [reply content/ReleaseNotes/TrueCommand/2.0-Beta.md]: Added

File: content/ReleaseNotes/TrueCommand/2.0-Beta.md (hunk 2 above)
Comment (jervin777) [reply content/ReleaseNotes/TrueCommand/2.0-Beta.md]: Changed

**Comment by Mrt134** [comment]
Manually verified both failing URLs as functioning properly

**Review by kmoore134 — DISMISSED** [review]

**Review by Mrt134 — APPROVED** [review]

**Comment by Mrt134** [comment]
Reconfirming URL fails are false positives - I'm approving and proceeding with the merge

**Pull request merged**